graphene_django/__init__.py

```diff
@@ -1,6 +1,6 @@
 from .types import DjangoObjectType
 from .fields import DjangoConnectionField
 
-__version__ = "2.3.0"
+__version__ = "2.3.2"
```

Comment:
Can you revert this? The version number will be incremented when we create a new release.

Reply:
I'll check. I didn't remember I changed version, could it be due to merge from some branch?